tests/test_helpers.py

```diff
@@ -515,6 +515,54 @@ def test_proxies_from_env_http_with_auth(mocker) -> None:
     assert proxy_auth.encoding == "latin1"
 
 
+# --------------------- get_env_proxy_for_url ------------------------------
+
+
+def test_get_env_proxy_for_url_when_url_matches_the_no_proxy_list(mocker) -> None:
+    mocker.patch("aiohttp.helpers.proxy_bypass", return_value=True)
+    mocker.patch(
+        "aiohttp.helpers.proxies_from_env",
+        return_value={"http": helpers.ProxyInfo("http://example.com", "")},
+    )
+    url = URL("http://aiohttp.io/path")
+    with pytest.raises(LookupError, match=r"Proxying is disallowed for `[^`]*`"):
+        proxy, proxy_auth = helpers.get_env_proxy_for_url(url)
```

Comment:
Assigning the results to variables never happens because the exception is raised before that. So it doesn't make sense to have those.
```suggestion
        helpers.get_env_proxy_for_url(url)
```